packages/core/tests/unit/models/goal-gauge-test.ts

```diff
@@ -10,40 +10,24 @@ module('Unit | Model | Gauge Visualization Fragment', function(hooks) {
   test('isValidForRequest', function(assert) {
     assert.expect(6);
 
-    let request = buildTestRequest([{ metric: 'rupees', parameters: {} }], []),
+    let request = buildTestRequest([{ field: 'rupees', parameters: {}, cid: 'cid_rupees' }], []),
       goalGauge = run(() =>
         run(() => this.owner.lookup('service:store').createRecord('all-the-fragments')).get('goalGauge')
       );
```

Comment:
Remove the `run` wrappers and add GoalGauge as the type in this fragment
https://github.com/yahoo/navi/blob/1b0d79a1e05921cd9f159f56fd30a2db191714e3/packages/core/tests/dummy/app/models/all-the-fragments.ts#L17
```suggestion
    let request = buildTestRequest([{ field: 'rupees', parameters: {}, cid: 'cid_rupees' }], []);
    const store = this.owner.lookup('service:store') as StoreService;
    const { goalGauge } = store.createRecord('all-the-fragments');
```